biomes.h

```diff
@@ -1496,8 +1496,8 @@ const UV BIOME_UVS[] {
 enum class MATERIAL : uint8_t {
   GRASS = 0,
   DIRT = 1,
-  STONE = 2,
-  ROCK = 3,
+  ROCK = 2,
+  STONE = 3,
```

Comment:
why swap the values here? Seems like if we're really using this like an enum, the actual integer values shouldn't matter. If they do matter, maybe an enum isn't the best fit? Or at least, maybe we should have a comment giving some context?

Reply:
the only reason it's swapped is because I tested ROCK material first, that's it
it doesn't really matter
this order matches the order of material texture path array in javascript :
https://github.com/webaverse/heightfield/pull/11/files#diff-dea54177dc473bc625fca043b2684efc953c68b615bffd13ef28f4d704078ab7L107